easier68k/core/opcodes/Bcc.py

```diff
@@ -364,4 +364,403 @@ def disassemble_instruction(cls, data: bytes):
     def from_str(cls, command: str, parameters: str):
         return bcc_from_str(command, parameters, BHI)
 
+
+class BLS(branch_code):
+    """
+    The BLS opcode compiles down to 63xx xxxx xxxx
+    The conditional here is C OR Z
+    """
+
+    cond_code = '\x03'
+
+    def conditional(self, simulator: M68K):
+        c = simulator.get_condition_status_code(ConditionStatusCode.C)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        if c or z:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BLS')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BLS, 3)
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BLS)
+
+
+class BCC(branch_code):
+    """
+    The BCC opcode compiles down to 64xx xxxx xxxx
+    The conditional here is NOT C
+    """
+
+    cond_code = '\x04'
+
+    def conditional(self, simulator: M68K):
+        c = simulator.get_condition_status_code(ConditionStatusCode.C)
+
+        if not c:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BCC')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BCC, 4)
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BCC)
+
+
+class BCS(branch_code):
+    """
+    The BCS opcode compiles down to 65xx xxxx xxxx
+    The conditional here is C
+    """
+
+    cond_code = '\x05'
+
+    def conditional(self, simulator: M68K):
+        c = simulator.get_condition_status_code(ConditionStatusCode.C)
+
+        if c:
```

Comment:
So `get_condition_status_code` returns a `bool`. We could just do: `return c` here.